docs/_docs/extras/deploying-with-mfa.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Deploying with MFA
+nav_order: 89
```

Comment:
Not sure where this should go. I'm guessing I'll have to update the other `nav_order` values too. 🤷‍♂

Reply:
Nope, you can leave the nav_order as-is.  Will figure out where it goes and the CI process will update it accordingly.